packages/forma-36-react-components/src/components/Asset/Asset.tsx

```diff
@@ -23,6 +24,36 @@ export function isAssetType(type: string): type is AssetType {
   return Object.keys(types).includes(type);
 }
 
+const renderImage = (src: string, title: string) => (
+  <React.Fragment>
+    <div className={styles['Asset__image-container']}>
+      <img
+        className={styles['Asset__image-container__image']}
+        src={src}
+        alt={title}
+      />
+    </div>
+    {title && (
+      <div className={styles['Asset__title-container']}>
+        <span className={styles['Asset__title-container__title']}>{title}</span>
+      </div>
+    )}
+  </React.Fragment>
+);
+
+const renderAsset = (type: AssetType, title: string) => {
+  return (
+    <div className={styles['Asset__asset-container']}>
+      <div className={styles['Asset__illustration-container']}>
+        <AssetIcon type={type} />
+      </div>
+      {title && (
+        <span className={styles['Asset__asset-container__title']}>{title}</span>
+      )}
+    </div>
+  );
+};
```

Comment:
Should we convert these render functions into functional components? They are just receiving two arguments and returning jsx so... wdyt?